### DISCUSSION TITLE
[Photoprism] Reverse proxy setup?

### DISCUSSION THREAD

**Discussion by MLNW** [tteck/Proxmox#978] [Q&A] [answered]
Hey, first of all great work on the script. The installation was a breeze.

Now I would like to make this service available to my larger network and eventually to the internet. For this purpose the authors recommend traefik but only provide an example of how to do so with the docker-compose method: https://docs.photoprism.app/getting-started/proxies/traefik/

Does anyone have an idea how to do this with the bare metal approach taken for the Photoprism installation?

**Comment by tteck** [comment, marked as answer]
You can run everything through Nginx Proxy Manager
- setup a free domain with duckdns.org (if you don't have a domain already)
- add proxy host in NPM
- edit newly created proxy host and setup Let's Encrypt SSL Certificates (SSL tab)

**Reply by MLNW** [reply]
That's exactly what I did. Thanks for the help!
